question_2.rb

```diff
@@ -0,0 +1,15 @@
+# Taking user input
+puts 'Enter Grade : '
+grade = gets.chomp.to_i
+
+
+# Conditionals statements to determine school level
+if grade >= 1 and grade <= 5
```

Comment:
instead of having two conditions you can handle using only one condition.

like grade <= 5, it will work.